src/randoop/ExpectedExceptionCheck.java

```diff
@@ -1,138 +1,76 @@
 package randoop;
 
-import java.io.ObjectStreamException;
-import java.lang.reflect.Modifier;
-
 import randoop.sequence.Execution;
 import randoop.util.Reflection;
 
 /**
- * A checker that checks for an expected exception from a method call.
+ * An {@code ExceptionCheck} that enforces the expectation of an exception
+ * or a subclass being thrown. In particular,
```

Comment:
For clarity, I would cut "or a subclass" here and move it three lines below.  That moves a detail out of the overview but makes the detailed explanation more accurate, so I think it improves them both.